staging/src/k8s.io/client-go/tools/remotecommand/remotecommand.go

```diff
@@ -52,91 +45,9 @@ type Executor interface {
 
 type streamCreator interface {
 	CreateStream(headers http.Header) (httpstream.Stream, error)
+	Run()
 }
 
 type streamProtocolHandler interface {
 	stream(conn streamCreator) error
 }
-
```

Comment:
I moved this code into a separate file so that it's easier to see shared types (in this file) vs SPDY-specific things (in `remotecommand_spdy.go`)